gpxsee: 11.6 -> 11.8

peterhoeg opened the pull request:
###### Description of changes

Add support for building with qt6 since gpxsee looks foreign with qt5 under KDE
anyway, we might as well start with qt6 that looks equally foreign.

I don't have access to anything other than linux x86_64 for testing, so leave
the qt5 option in here for now.

Changelog: https://build.opensuse.org/package/view_file/home:tumic:GPXSee/gpxsee/gpxsee.changes

Does it work? If I don't get lost this weekend while MTB'ing, it should be fine.

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

erictapen committed:
python3Packages.python-mapnik: unstable-2020-02-24 -> unstable-2020-09-08

Also:

- use libxml2 for xml parsing
- use newest boost
- patch to find libmapnik{json,wkt}.a
- fix license
- add erictapen as maintainer
- use same harfbuzz as mapnik
- use pytestCheckHook, disable failing tests

Co-Authored-by: Robert Schütz <nix@dotlambda.de>

erictapen committed:
mapnik: 3.1.0 -> unstable-2022-04-14

- use CMake. I just didn't got it to work with SCons.
- patch cmake to find harfbuzz
- prefetch catch2 for enabling tests
- add erictapen as maintainer
- build mapnik-config with scons, as it isn't build by cmake
- use python3 exclusively
- build with harfbuzz-icu
- change license to lgpl21Plus
- use PROJ 8
- use latest scons, but use cmake for everything but mapnik-config
- don't make postgresql optional

Co-Authored-by: Robert Schütz <nix@dotlambda.de>

somasis committed:
linja-pi-pu-lukin: init at unstable-2021-10-29

lovesegfault committed:
linkFarm: add entries to passthru

lovesegfault committed:
linkFarm: make last entry win in case of list repeats

lovesegfault committed:
linkFarm: allow entries to be an attrset

dotlambda committed:
python310Packages.pysvn: 1.9.12 -> 1.9.18

dotlambda committed:
python310Packages.Wand: run tests

r-ryantm committed:
isolyzer: 1.3.0 -> 1.4.0

dotlambda committed:
rymdport: init at 3.2.0

dotlambda committed:
gdal: 3.4.2 -> 3.5.2

https://github.com/OSGeo/gdal/blob/v3.5.2/NEWS.md

dotlambda committed:
libgeotiff: 1.5.1 -> 1.7.1

https://github.com/OSGeo/libgeotiff/blob/1.7.1/libgeotiff/NEWS

dotlambda committed:
brunsli: init at 0.1

dotlambda committed:
lerc: init at 3.0

dotlambda committed:
gplates: use python39

Boost 1.75 is not compatible with Python 3.10.

dotlambda committed:
python310Packages.fiona: fix tests with GDAL 3.5.1

onny committed:
keeperrl: alpha28 -> alpha34

Majiir committed:
nut: fix search modes not finding dynamic libs

somasis committed:
sitelen-seli-kiwen: init at unstable-2022-06-28

kekrby committed:
nixos/modules/config/gtk/gtk-icon-cache: do not generate icon caches for files in $out/share/icons

Icon caches should be generated for directories under $out/share/icons, as the comments mention, but it is done for files as well which does not make the build fail but results in files under $out/share/icons being turned into empty directories, see /run/current-system/sw/share/icons/gnome-logo-text.svg as an example.

pacien committed:
matrix-appservice-discord: 1.0.0 -> 3.1.0

- This also takes into account the transfer of the project and repository
  to the matrix-org organisation on GitHub.

- The Node version assertion has been removed because our `pkgs.nodejs`
  already points to version 18.

- This replaces the `.nix` lock file (generated with `yarn2nix`) with a
  hash-pinned fixed-output derivation, which is simpler to work with.

- This replaces the old "generate.sh" script with one that updates all the
  version info and hashes (package.json, source version, source hash,
  and Yarn dependencies hash).
  This is inspired by the update script used for the
  "matrix-appservice-slack" package.

- This switches to srcOnly for node sources.

- Added pkgs.matrix-sdk-crypto-nodejs new required native dependency.

divanorama committed:
jemalloc: fix aarch64-darwin build

Switch to default clang11 and apply a patch to fix tests.

The issue is compiler optimizations for unused `malloc` calls
being applied to jemalloc tests with empty `--with-jemalloc-prefix=`.
Tests want to test bad arguments to malloc and optimization broke
such assertions in some of tests, so patch is applied to disable
builtin functions optimizations (only for tests code).

Fixes https://github.com/NixOS/nixpkgs/issues/152056

Patch submission upstream
https://github.com/jemalloc/jemalloc/pull/2340

fabaff committed:
python310Packages.ytmusicapi: 0.22.0 -> 0.24.0

B4rc1 committed:
ulauncher: 5.12.1 -> 5.15.0

wegank committed:
gupnp-*: add darwin support

wegank committed:
dleyna-*: add darwin support

r-ryantm committed:
xmlada: 22.0.0 -> 23.0.0

eigengrau committed:
nixos/zsh: don’t be noisy when scripts are run with -u

When a script specifies the shell option “nounset” as part of the shebang (e.g.,
via “#!/usr/bin/env -S zsh -u”), our initialization scripts would produce error
messages of the form:

    __ETC_FOO_SOURCED: parameter not set

These messages could probably be confusing to users when running such scripts.
By providing a fall-back in the parameter expansion, we can avoid them.

This patch does not address interactive shell start-up, where such messages may
(or may not) be less problematic.

gador committed:
python3Packages.reorder-python-imports: 3.8.1 -> 3.8.5

Signed-off-by: Florian Brandes <florian.brandes@posteo.de>

r-ryantm committed:
haste-server: 787d839086faee0b55c76ce5944fa207d22e85b4 -> 20919c946602b8151157f647e475e30687a43727

eigengrau committed:
nixos/shells: support defining aliases beginning with a dash or plus

Both Zsh and Bash support aliases that begin with characters also used to
indicate options to the “alias” built-in command, as long as the alias
definition is preceeded by a double dash.

This allows, e.g, for “alias -- +x=chmod +x”.

risicle committed:
kea: fix build on darwin

risicle committed:
brunsli: tidy patching

risicle committed:
brunsli: fix building on darwin

risicle committed:
gdal: fix build on darwin

Infinidoge committed:
packwiz: unstable-2022-09-25 -> unstable-2022-10-29

dotlambda committed:
gdal: skip test failing when using PROJ < 8

gador committed:
python3Packages.reorder-python-imports: 3.8.5 -> 3.9.0

Signed-off-by: Florian Brandes <florian.brandes@posteo.de>

gador committed:
python3Packages.pipenv-poetry-migrate: 0.2.0 -> 0.2.1

Changelog:
https://github.com/yhino/pipenv-poetry-migrate/releases/tag/v0.2.1

Signed-off-by: Florian Brandes <florian.brandes@posteo.de>

gador committed:
python3Packages.apsw: 3.39.3.0 -> 3.39.4.0

https://rogerbinns.github.io/apsw/changes.html

Signed-off-by: Florian Brandes <florian.brandes@posteo.de>

CRTified committed:
nixos/adguardhome: allow for empty/unmanaged configs

This commit fixes broken non-declarative configs by
making the assertions more relaxed.
It also allows to remove the forced configuration merge by making
`settings` `null`able (now the default).

Both cases (trivial non-declarative config and `null`able config) are
verified with additional tests.

Fixes #198665

dotlambda committed:
mapnik: unstable-2022-04-14 -> unstable-2022-10-18

SuperSandro2000 committed:
onefetch: 2.12.0 -> 2.13.2

Izorkin committed:
nixos/peertube: add hsts header to nginx configuration

Izorkin committed:
nixos/peertube: add quic header to nginx configuration

Izorkin committed:
nixos/peertube: fix start services

fgaz committed:
git-credential-keepassxc: init at 0.10.1

siriobalmelli committed:
ipcalc: add patch to disable tests failing in Darwin sandbox

Signed-off-by: Sirio Balmelli <sirio@b-ad.ch>

chuangzhu committed:
linuxPackages.rtl8723ds: init at unstable-2022-10-20

Fuuzetsu committed:
rustBuildCrate: properly handle cargo env pragmas with spaces

There are two problems: first that we end up splitting on spaces in the
loop. Even when that is fixed, we still would split on spaces in the
`export` inside the loop. We need to guard against both.

Fixes #199298

Confirmed that it fixes the case mentioned in the ticket:

```console
[nix-develop]$ $(nix-build -I nixpkgs=/home/shana/programming/nixpkgs Cargo.nix -A rootCrate.build  --no-out-link)/bin/nix-rustc-env-escape-repro
Expecting three words, got: first second third
```

I think this is going to cause a rebuild of every Rust package even if
they were unaffected, not much we can do here.

symphorien committed:
nixos/power-management: fix deadlock with post-resume.{target,service}

Fixes a deadlock where post-resume.target has After=post-resume.service
and post-resume.service runs systemctl try-restart post-resume.target
the systemctl call cannot complete if post-resume.target was already
queued at that time.

r-ryantm committed:
melt: 0.4.1 -> 0.5.0

r-ryantm committed:
soft-serve: 0.4.0 -> 0.4.1

r-ryantm committed:
cava: 0.8.2 -> 0.8.3

r-ryantm committed:
minikube: 1.27.1 -> 1.28.0

tpwrules committed:
zfp: 0.5.5 -> 1.0.0, fix issues

Remove examples as they have some common executable names likely to
cause conflicts, there is no way to properly install them, and they are
not particularly useful. Remove now-unnecessary installation hacks.

Disable the long tests as they are just bigger versions of the short
tests and end up being pretty slow.

Build CUDA kernels for all GPUs to avoid issues at runtime.

tjni committed:
python310Packages.tox: 3.26.0 -> 3.27.1

figsoda committed:
onefetch: minor improvements, add figsoda as a maintainer

* add back the zstd-pkg-config patch
* move cmake to nativeBuildInputs
* fix test instead of skipping it

mweinelt committed:
python3Packages.gpiozero: disable on darwin

sifmelcara committed:
krita: 5.1.1 -> 5.1.3

Luflosi committed:
yosys: 0.22 -> 0.23

https://github.com/YosysHQ/yosys/releases/tag/yosys-0.23
The protobuf backend was removed in https://github.com/YosysHQ/yosys/pull/3508.

veehaitch committed:
release-notes: mention new `services.github-runners` & breaking changes

r-ryantm committed:
icinga2: 2.13.5 -> 2.13.6

lorenz committed:
dsview: 1.1.2 -> 1.2.1

r-ryantm committed:
openturns: 1.19 -> 1.20

vbgl committed:
ocamlPackages.qcheck: 0.19.1 → 0.20

dotlambda committed:
proj: 9.0.0 -> 9.1.0

dotlambda committed:
python310Packages.pyproj: 3.3.1 -> 3.4.0

dotlambda committed:
python310Packages.basemap: 1.3.4 -> 1.3.6

https://github.com/matplotlib/basemap/blob/v1.3.6/CHANGELOG.md

dotlambda committed:
proj: don't add cURL dependency to CMake config

teutat3s committed:
redis-desktop-manager: 0.9.1 -> 2022.5, rename to RESP.app

RESP.app - GUI for Redis (Formerly RedisDesktopManager)
"RESP.app is joining forces with Redis to offer the Redis community the
best possible developer experience"
See: https://github.com/RedisInsight/RedisDesktopManager
and
https://redis.com/blog/respapp-joining-redis/

dotlambda committed:
python310Packages.graphviz: 0.20 -> 0.20.1

https://github.com/xflr6/graphviz/blob/0.20.1/CHANGES.rst

dotlambda committed:
python310Packages.keyring: 23.9.3 -> 23.11.0

https://github.com/jaraco/keyring/blob/v23.11.0/CHANGES.rst

dotlambda committed:
gdal: don't depend on kea

The actual (optional) dependency is http://www.kealib.org/.

PedroHLC committed:
linux-lqx: 6.0.7-lqx1 -> 6.0.8-lqx1

PedroHLC committed:
linux-zen: 6.0.7-zen1 -> 6.0.8-zen1

collares committed:
python310Packages.cypari2: 2.1.2 -> 2.1.3

sikmir committed:
sic: 1.2 → 1.3

dotlambda committed:
mopidy-ytmusic: use ytmusicapi 0.22.0

r-ryantm committed:
redli: 0.6.0 -> 0.7.0

r-ryantm committed:
sympa: 6.2.68 -> 6.2.70

r-ryantm committed:
victoriametrics: 1.83.0 -> 1.83.1

r-ryantm committed:
python310Packages.libsass: 0.21.0 -> 0.22.0

sternenseemann committed:
gprbuild, gprbuild-boot: 22.0.0 -> 23.0.0

sternenseemann committed:
gnatcoll-core: 22.0.0 -> 23.0.0

sternenseemann committed:
gnatcoll-{sql*,postgres,xref,db2ada}, gnatinspect: 22.0.0 -> 23.0.0

sternenseemann committed:
gnatcoll-{lzma,gmp,iconv,omp,python3,readline,syslog,zlib}: 22.0.0 -> 23.0.0

winterqt committed:
matrix-appservice-discord: run pre/post hooks for overridden phases

lopsided98 committed:
samba: fix cross-compilation

The last update mistakenly removed wafHook and the patch required to make cross-
compilation work. In addition, the bundled version of heimdal is now too new to
work with asn1_compile from the heimdal package in nixpkgs (it isn't out of
date, there just hasn't been a release in years and samba uses an unreleased
version).

To fix the latter issue, the native build of samba saves asn1_compile and
compile_et from the bundled heimdal into the dev output, allowing them to be
available at build time when cross-compiling.

AndersonTorres committed:
ares: 129 -> 130.1

AndersonTorres committed:
dialog: 1.3-20211214 -> 1.3-20220728

AndersonTorres committed:
doublecmd: refactor to new overlay-style overridable attributes

AndersonTorres committed:
fceux: refactor to new overlay-style overridable attributes

AndersonTorres committed:
desmume: 0.9.11+unstable=2021-09-22 -> 0.9.13

dotlambda committed:
python310Packages.wand: rename from Wand

samuela committed:
python3Packages.wandb: 0.13.4 -> 0.13.5

trofi committed:
synergy: use xorg.* packages directly instead of xlibsWrapper indirection

Validated as no change in `out` output with `diffoscope`.

fpletz committed:
batman-adv: 2022.1 -> 2022.3

sternenseemann committed:
rure: 0.2.1 -> 0.2.2

sternenseemann committed:
gnatinspect, gnatcoll-db2ada: drop gnatcoll- prefix from pname

AndersonTorres committed:
desmume: mark as broken on linux-aarch64

wegank committed:
grilo: add darwin support

AndersonTorres committed:
asl: 142-bld211 -> 142-bld232

mweinelt committed:
CONTRIBUTING: Reference release notes in package bumps

This change improves the recommendation for good commit messages to
include release notes on package bumps.

Including the release notes increases the likelihood that they will be
taken into consideration during review. Having them included in the
review is important to be able to judge whether changes made during a
version bump are sensible, sufficient or complete.

The burden of retrieving the release notes for arbitrary package bumps
should not rest on the relatively small group of reviewers.

Signed-off-by: Martin Weinelt <hexa@darmstadt.ccc.de>

K900 committed:
kde-frameworks: 5.99 -> 5.100

K900 committed:
ludusavi: fix eval

K900 committed:
tela-circle-icon-theme: fix eval

K900 committed:
extra-cmake-modules: drop merged patch

K900 committed:
karchive: add new dependency

bjornfor committed:
qownnotes: update hash to upstream provided format

So that we can get the checksum from upstream with the curl command in
the comment.

No rebuild.

bjornfor committed:
qownnotes: 22.11.4 -> 22.11.5

Changelog: https://github.com/pbek/QOwnNotes/blob/develop/CHANGELOG.md#22115

zowoq committed:
containerd: 1.6.9 -> 1.6.10

https://github.com/containerd/containerd/releases/tag/v1.6.10

ryantm committed:
discourse: 2.9.0.beta11 -> 2.9.0.beta12

cpcloud committed:
duckdb: 0.5.1 -> 0.6.0

rhoriguchi committed:
gnomeExtensions: auto-update

figsoda committed:
cargo-edit: 0.11.5 -> 0.11.6

figsoda committed:
cargo-auditable: 0.5.3 -> 0.5.4

figsoda committed:
rust-audit-info: 0.5.1 -> 0.5.2

azahi committed:
mautrix-telegram: fix build

ZHF: #199919

trofi committed:
libstroke: use xorg.* packages directly instead of xlibsWrapper indirection

Validated as no change in `out` output with `diffoscope`.

figsoda committed:
ruff: 0.0.118 -> 0.0.119

r-ryantm committed:
gh-dash: 3.4.2 -> 3.5.1

foo-dogsquared committed:
moar: init at 1.10.0

figsoda committed:
vimPlugins.nvim-treesitter: move grammar generation from fetch to grammar.nix

dotlambda committed:
python310Packages.pikepdf: 6.2.2 -> 6.2.4

https://github.com/pikepdf/pikepdf/blob/v6.2.4/docs/releasenotes/version6.rst

r-ryantm committed:
icewm: 3.2.1 -> 3.2.2

K900 committed:
krunner-pass: fix cmake configuration

K900 committed:
kmplayer: fix cmake configuration

K900 committed:
ksmoothdock: patch out -Werror

K900 committed:
bcompare: 4.4.2.26348 -> 4.4.4.27058

austinbutler committed:
openshot-qt: fix Python 3.10 incompatibility

mweinelt committed:
python3Packages.datasets: fix build after dill bump

r-ryantm committed:
python310Packages.aiopvapi: 2.0.3 -> 2.0.4

zowoq committed:
nil: 2022-11-07 -> 2022-11-15

https://github.com/oxalica/nil/releases/tag/2022-11-15

romildo committed:
xfce.xfce4-settings: 4.16.4 -> 4.16.5

r-ryantm committed:
libplctag: 2.5.3 -> 2.5.4

r-ryantm committed:
libreddit: 0.23.1 -> 0.24.0

AndersonTorres committed:
jwasm: add changelog page

Also, use the new overlay-style overridable attributes.

AndersonTorres committed:
jwasm: 2.15 -> 2.16

jqqqqqqqqqq committed:
nixos/nginx: fix default listen port options

r-ryantm committed:
appthreat-depscan: 2.2.1 -> 2.3.0

prusnak committed:
kubecolor: 0.0.20 -> 0.0.21

this also changes the upstream
from hidetatz/kubecolor
to kubecolor/kubecolor

AndersonTorres committed:
netatalk: refactor

wyndon committed:
protonvpn-gui: add meta.mainProgram

r-ryantm committed:
brev-cli: 0.6.170 -> 0.6.176

ajs124 committed:
jenkins: 2.361.3 -> 2.361.4

https://www.jenkins.io/changelog-stable//#v2.361.4

lovesegfault committed:
tests.trivial-builders.linkFarm: init

dotlambda committed:
python310Packages.aprslib: 0.7.1 -> 0.7.2

figsoda committed:
ruff: 0.0.119 -> 0.0.120

figsoda committed:
lunatic: 0.10.1 -> 0.12.0

prusnak committed:
python3Packages.latexify-py: init at 0.2.0

tjni committed:
cargo-raze: 0.12.0 -> 0.16.0

tjni committed:
python310Packages.dill: add passthru tests

Adds apache-beam and datasets to be tested when dill is updated. These
two packages broke when updating dill from 0.3.5.1 to 0.3.6.

kevincox committed:
mautrix-facebook: unstable-2022-05-06 → 0.4.1

squalus committed:
librewolf: 106.0.3-1 -> 107.0-1

SuperSandro2000 committed:
mpv-unwrapped: 0.34.1 -> 0.35.0

SuperSandro2000 committed:
addlicense: unstable-2021-04-22 -> 1.1.0

techknowlogick committed:
consul: 1.13.3 -> 1.14.0

SuperSandro2000 committed:
yq-go: 4.30.1 -> 4.30.4

whentze committed:
terraria-server: 1.4.4.2 -> 1.4.4.8.1

yurket committed:
d-seams: fix failing builds

cxxopts failing to build with clang11:

  /build/source/src/include/external/cxxopts.hpp:488:42: error: no member named 'numeric_limits' in namespace 'std'
            if (u > static_cast<U>(-std::numeric_limits<T>::min()))

Use the patch from upstream, which updates cxxopts 2.2.0 -> 3.0.0

dotlambda committed:
python310Packages.huawei-lte-api: 1.6.6 -> 1.6.7

ncfavier committed:
dwarf-therapist: restrict platforms to x86

Avoid trying to build for aarch64 on Hydra. Dwarf Fortress only runs on x86.

figsoda committed:
ruff: 0.0.120 -> 0.0.121

dotlambda committed:
python310Packages.miniaudio: 1.54 -> 1.55

https://github.com/irmen/pyminiaudio/releases/tag/v1.55

ElvishJerricco committed:
Revert "nixos: Fix fsck with systemd 251.6 and later"

This reverts commit d9b1bde390eb133a3da66c8abd902ea2b754938c.

ElvishJerricco committed:
nixos: Add util-linux to systemd PATH to fix fsck with systemd 251.6

This is an alternative to the reverted
d9b1bde390eb133a3da66c8abd902ea2b754938c, which broke systemd stage 1

SuperSandro2000 committed:
python310Packages.pushover-complete: enable tests

SuperSandro2000 committed:
python310Packages.ibm-watson: remove not required tox requirement

iNode committed:
micronaut: 1.3.7 -> 3.7.2 (#197945)

resolve https://github.com/NixOS/nixpkgs/issues/160676

SuperSandro2000 committed:
Revert "gplates: fix failing build"

This reverts commit 8a236e246ab1aceea6e5b181f0d566be6bb3a614.

r-ryantm committed:
datree: 1.7.3 -> 1.8.0

PhotonQuantum committed:
vimPlugins.barbecue-nvim: init at 2022-11-16

PhotonQuantum committed:
maintainers: add lightquantum

dotlambda committed:
libdeltachat: 1.100.0 -> 1.101.0

https://github.com/deltachat/deltachat-core-rust/blob/1.101.0/CHANGELOG.md

Commit:
vimPlugins: update

romildo committed:
matcha-gtk-theme: 2022-06-07 -> 2022-11-15 (#201410)

fpletz committed:
Revert "unifi-poller: 2.1.3 -> 2.1.7"

This reverts commit 2efa9c8133d221b96e46fb17e7ff668068968500.

Fixes #201388.

bobby285271 committed:
pantheon.elementary-calendar: 6.1.1 -> 6.1.2

bobby285271 committed:
pantheon.elementary-files: 6.2.0 -> 6.2.1

bobby285271 committed:
pantheon.elementary-photos: 2.7.5 -> 2.8.0

bobby285271 committed:
pantheon.elementary-greeter: 6.1.0 -> 6.1.1

happysalada committed:
surrealdb: init at 1.0.0-beta.8

bobby285271 committed:
pantheon.wingpanel-applications-menu: 2.10.2 -> 2.11.0

bobby285271 committed:
touchegg: 2.0.14 -> 2.0.15

r-ryantm committed:
ferium: 4.2.1 -> 4.2.2

zowoq committed:
cri-o: refactor wrapper

zowoq committed:
buildah: refactor wrapper

- also move `preferLocalBuild` out of let..in

zowoq committed:
podman: refactor wrapper

aaronjheng committed:
copper: mark as broken

viraptor committed:
doggo: enable darwin support

aaronjheng committed:
coyim: tweak platforms

aaronjheng committed:
gatling: mark as broken

r-ryantm committed:
goeland: 0.11.0 -> 0.12.1

figsoda committed:
ruff: 0.0.121 -> 0.0.122

github-actions[bot] committed:
terraform-providers.cloudflare: 3.27.0 → 3.28.0

github-actions[bot] committed:
terraform-providers.newrelic: 3.7.0 → 3.7.1

github-actions[bot] committed:
terraform-providers.alicloud: 1.191.0 → 1.192.0

github-actions[bot] committed:
terraform-providers.tencentcloud: 1.78.10 → 1.78.11

zowoq committed:
terraform-providers.baiducloud: 1.18.0 -> 1.18.2

foo-dogsquared committed:
licenses: add bsd2WithViews

r-ryantm committed:
imgui: 1.88 -> 1.89

figsoda committed:
vimPlugins.nvim-treesitter: check queries in passthru.tests

equirosa committed:
signal-desktop: 5.63.0 -> 5.63.1

aaronjheng committed:
isrcsubmit: mark as broken

aaronjheng committed:
lazpaint: mark as broken

r-ryantm committed:
jwx: 2.0.6 -> 2.0.7

sikmir commented:
Result of `nixpkgs-review pr 201440` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>gpxsee-qt6</li>
    <li>qt6Packages.qtpbfimageplugin</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gpxsee (gpxsee-qt5)</li>
  </ul>
</details>

coopbri committed:
nixos/misc: fix grammar typo

tjni committed:
python310Packages.bibtexparser: clean up python 2 dependencies (#201457)

Latest version no longer depends on unittest2 and future packages, so we
can clean them up.

ghuntley committed:
mastodon: add ghuntley as maintainer

peterhoeg committed:
psst: unstable-2022-05-19 -> 2022-10-13

picnoir committed:
nsncd: unstable-2021-10-20 -> unstable-2022-11-14

This bump fixes the response format for IPV6-only requests. The bogus
response message was crashing glibc.

See https://github.com/nix-community/nsncd/pull/1 for more informations.

peterhoeg commented:
@GrahamcOfBorg build gpxsee-qt5 gpxsee-qt6

ncfavier committed:
remove dead `make-symlinks` builder

Introduced in 5808ac7148850a79d9ff59e37368bdd13f187adf, never used again
after b06335a835ac8b7bc40c77e0089e8e8556bac3cc.

ncfavier committed:
dwarf-therapist: pass through `meta`

sternenseemann committed:
ocamlPackages.ocaml_extlib: 1.7.8 -> 1.7.9

Use new dune based build system. This no longer has a way to
enable the non-minimal build. As it turns out, though, no
package required that (and it was impossible to get a non-minimal
extlib build from OPAM before anyways).

The old expression needs to be retained for extlib 1.7.7 and just
moved over.

sternenseemann committed:
ocamlPackages.extlib: rename from ocaml_extlib

This matches the name used in dune and on OPAM.

r-ryantm committed:
meerk40t: 0.8.0031 -> 0.8.1000

r-ryantm committed:
meilisearch: 0.29.1 -> 0.29.2

ciferkey committed:
adw-gtk3: 4.0 -> 4.1

r-ryantm committed:
netbird: 0.10.7 -> 0.10.8

r-ryantm committed:
oh-my-posh: 12.13.0 -> 12.13.3

dotlambda committed:
python310Packages.check-manifest: fix tests

dotlambda committed:
lisgd: 0.3.5 -> 0.3.6

r-ryantm committed:
clojure: 1.11.1.1189 -> 1.11.1.1200

schuelermine committed:
vscode-extensions.matangover.mypy: init at 0.2.2

dotlambda committed:
python310Packages.git-annex-adapter: fix tests

schuelermine committed:
vscode-extensions.thenuprojectcontributors.vscode-nushell-lang: init at 0.7.0

r-ryantm committed:
operator-sdk: 1.25.1 -> 1.25.2

schuelermine committed:
vscode-extensions.bmalehorn.vscode-fish: init at 1.0.31

r-ryantm committed:
oxker: 0.1.6 -> 0.1.7

r-ryantm committed:
pass2csv: 0.3.2 -> 1.0.0

r-ryantm committed:
picom: 9.1 -> 10

thiagokokada committed:
nixos/picom: remove experimentalBackends option

Removed by upstream in the recent v10 release.

thiagokokada committed:
nixos/docs: document picom module changes

dasJ committed:
samba4: 4.17.2 -> 4.17.3

lovesegfault committed:
linux: 5.10.154 -> 5.10.155

lovesegfault committed:
linux: 5.15.78 -> 5.15.79

lovesegfault committed:
linux: 6.0.8 -> 6.0.9

r-ryantm committed:
radioboat: 0.2.2 -> 0.2.3

r-ryantm committed:
rootlesskit: 1.0.1 -> 1.1.0

github-actions[bot] committed:
terraform-providers.digitalocean: 2.23.0 → 2.24.0

github-actions[bot] committed:
terraform-providers.fastly: 2.4.0 → 3.0.0

github-actions[bot] committed:
terraform-providers.google: 4.43.0 → 4.43.1

github-actions[bot] committed:
terraform-providers.google-beta: 4.43.0 → 4.43.1

github-actions[bot] committed:
terraform-providers.opentelekomcloud: 1.31.7 → 1.31.8

github-actions[bot] committed:
terraform-providers.oci: 4.99.0 → 4.100.0

github-actions[bot] committed:
terraform-providers.vault: 3.10.0 → 3.11.0

r-ryantm committed:
ssh-to-age: 1.0.2 -> 1.0.3

rapenne-s commented:
Result of `nixpkgs-review pr 201440` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>gpxsee (gpxsee-qt5)</li>
    <li>gpxsee-qt6</li>
    <li>qt6Packages.qtpbfimageplugin</li>
  </ul>
</details>

rapenne-s commented:
qt5 and qt6 flavors both work for me, I've been able to load a lot of GPX at the same time, everything works for on x86_64-linux at least

peterhoeg commented:
Thanks for checking @rapenne-s.

@sikmir & @womfoo as maintainers - any concerns about this? 

FYI, I have tested both qt5 and qt6 as well and both are working.

sikmir commented on pkgs/applications/misc/gpxsee/default.nix:
```diff
@@ -19,14 +36,18 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ qmake qttools wrapQtAppsHook ];
 
-  preConfigure = ''
```
Why you removed translations?

sikmir commented:
11.9 is out meanwhile.

peterhoeg committed:
WIP

peterhoeg commented:
preConfigure blows up with qt6, but just for my understanding, how can you tell that the translations are gone? Do you set the language via the LANG env var?

FYI, the qt6 issue is separate and I’ll open an issue for that.

peterhoeg committed:
WIP

flokli commented:
Closing due to mass-ping. Please open a new PR.

sikmir commented:
> preConfigure blows up with qt6, but just for my understanding, how can you tell that the translations are gone? Do you set the language via the LANG env var?

@peterhoeg Yes, if your system `LANG=en`, you can run with different locale `LANG=fi gpxsee`. Or just check `$out/share/gpxsee/translations` folder, it should contain actual translation files.
